clients/src/main/java/org/apache/kafka/clients/Metadata.java

```diff
@@ -248,6 +250,13 @@ public synchronized long lastSuccessfulUpdate() {
     }
 
     /**
+     * The max allowable age of metadata.
+     */
+    public synchronized long getMetadataMaxAge() { 
```

Comment:
Would it be good to rename this to : getMetadataMaxAllowedAge() or something similar.